inc/ticket.class.php

```diff
@@ -5331,15 +5333,22 @@ static function showCentralList($start, $status = "process", $showgrouptickets =
                   break;
 
                case "process" :
-                  $options['criteria'][0]['field']      = 12; // status
-                  $options['criteria'][0]['searchtype'] = 'equals';
-                  $options['criteria'][0]['value']      = 'process';
-                  $options['criteria'][0]['link']       = 'AND';
 
-                  $options['criteria'][1]['field']      = 8; // groups_id_assign
-                  $options['criteria'][1]['searchtype'] = 'equals';
-                  $options['criteria'][1]['value']      = 'mygroups';
-                  $options['criteria'][1]['link']       = 'AND';
+                  $options['criteria'][2]['field']      = 8; // groups_id_assign
+                  $options['criteria'][2]['searchtype'] = 'equals';
+                  $options['criteria'][2]['value']      = 'mygroups';
+                  $options['criteria'][2]['link']       = 'AND';
+
+                  $options['criteria'][4]['link']                       = 'AND';
+                  $options['criteria'][4]['criteria'][0]['link']        = 'AND';
+                  $options['criteria'][4]['criteria'][0]['field']       = 12;
+                  $options['criteria'][4]['criteria'][0]['searchtype']  = 'equals';
+                  $options['criteria'][4]['criteria'][0]['value']       = 1;
+
+                  $options['criteria'][4]['criteria'][1]['link']        = 'OR';
+                  $options['criteria'][4]['criteria'][1]['field']       = 12;
+                  $options['criteria'][4]['criteria'][1]['searchtype']  = 'equals';
+                  $options['criteria'][4]['criteria'][1]['value']       = 'process';
```

Comment:
Why using indexes 2 and 4 ?
Could it be possible to have something like
```php
$options['criteria'] = [
   [
      'field' => 8
      ...
   ],
   [
      'link' => 'AND',
      'criteria' => [
         [
            'field' => 12,
            ...
         ],
         [
            'field' => 12,
            ...
         ]
      ]
   ]
];
```

Also, position of AND/OR links seems weird to me, but maybe it is done like this in search class.

Reply:
Fixed by using new convention